pkg/feeder/feeder.go

```diff
@@ -657,14 +661,13 @@ func (c Client) CallEstimateFeeWithBody(blockIdentifier map[string]string, reqBo
 	req, err := c.newRequest(
 		"POST", "/estimate_fee", blockIdentifier, reqBody)
 	if err != nil {
-		Logger.With("Error", err, "Gateway URL", c.BaseURL).Error("Unable to create a request for estimate_fee.")
+		c.logger.Errorw("Unable to create a request for estimate_fee.", "error", err, "url", c.BaseURL)
 		return nil, err
 	}
 	var res EstimateFeeResponse
 	_, err = c.do(req, &res)
 	if err != nil {
-		Logger.With("Error", err, "Gateway URL", c.BaseURL).
-			Error("Error connecting to gateway.")
+		c.logger.Errorw("Error connecting to gateway.", "error", err, "url", c.BaseURL)
```

Comment:
```suggestion
		c.logger.Errorw("Error connecting to gateway.", "error", err, "url", c.BaseURL)
```

Reply:
It appears no change is suggested--I tried to remove the rest of the periods from the log messages in a recent commit.